sniffles/regex_generator.py

```diff
@@ -139,7 +139,7 @@ def create_regex_list(number, lambd, type_dist, char_dist, class_dist,
                 myregex += o
         # check if this compiles or not
         if not check_pcre_compile(myregex):
-            print("pcre compile failed ... continuing")
+            # print("pcre compile failed ... continuing")
```

Comment:
Wouldn't it be better to delete this line or to replace with an actual comment, than to leave it as a commented-out code?